Maps/MapWriter.cpp

```diff
@@ -100,20 +100,14 @@ namespace MapWriter {
 		{
 			WriteContainerSize(streamWriter, tileInfos.size());
 
-			for (TileInfo tileInfo : tileInfos)
-			{
-				streamWriter.Write(&tileInfo, sizeof(TileInfo));
-			}
+			streamWriter.Write(tileInfos.data(), sizeof(TileInfo) * tileInfos.size());
```

Comment:
Yeah, I didn't know how to use the data function attached to vector and string at the time of writing this code, causing the less efficient loop. 

I haven't thought about standardizing calculating the size of a container. It doesn't matter to me, but since you have a preference, I'll try to file away putting the size() first.